src/components/Console.jsx

```diff
@@ -49,13 +49,19 @@ export default function Console({
         className="label console__label"
         onClick={partial(onToggleVisible, currentProjectKey)}
       >
-        Console
-        <span className="console__chevron u__icon">{chevron}</span>
+        <div>
```

Comment:
In response to your question, I think it’s fine to not give this a class name if we don’t need to style it!